### PR TITLE
Sessions overview

### PR DESCRIPTION
**Pull request by antonio-macovei**
I updated the sessions overview according to this year names and structure.

### CODE REVIEW

File: 02-access-control/README.md [hunk 1]
```diff
@@ -270,29 +270,29 @@ Alternatively, you can see the entire HTML code of a web page by selecting View
...
+* **Elements**: In this tab you can see the HTML structure of the page. On the right panel, you can see the styles applied to each element when selected and add, remove or edit the properties directly from there. This kind of inspection could lead to the discovery of hidden elements which can be toggled into view by altering the CSS code or could lead to the discovery of commented pieces of code which could contain sensitive data. Also, the [DOM](https://github.com/hexcellents/sss-web/wiki/Session-01:-access-control-&-Browser-Security-Model#dom-document-object-model) (Document Object Model) structure of the page can be altered, and elements can be added or removed, such as scripts, input fields, etc. (any element in fact), which means that any JavaScript code used to sanitize user input or perform other functions can be bypassed.
```
Comment (carrrina) [inline 02-access-control/README.md:273]: This shouldn't be modified

**Review by carrrina — CHANGES_REQUESTED** [review]

File: 02-access-control/README.md (hunk 1 above)
Comment (antonio-macovei) [reply 02-access-control/README.md:273]: Fixed

**Comment by gabrielmocanu** [comment]
@carrrina @antonio-macovei 

What is the status of this?
When this can be merged? I saw there is a conflict, probably it will need a rebase before.

File: 09-privilege-escalation/README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+- https://github.com/swisskyrepo/PayloadsAllTheThings/blob/master/Methodology%20and%20Resources/Linux%20-%20Privilege%20Escalation.md
+- https://book.hacktricks.xyz/linux-unix/privilege-escalation
+- https://dirtycow.ninja/
+- https://github.com/lucyoa/kernel-exploits
+- https://github.com/offensive-security/exploitdb-bin-sploits/tree/master/bin-sploits
+- https://gtfobins.github.io/
+- https://betterprogramming.pub/escaping-docker-privileged-containers-a7ae7d17f5a1
+- https://github.com/carlospolop/privilege-escalation-awesome-scripts-suite/tree/master/linPEAS
+- https://app.hackthebox.eu/machines
+# Challenges
```
Comment (carrrina) [inline 09-privilege-escalation/README.md:140]: Add a new line before this

**Review by carrrina — CHANGES_REQUESTED** [review]

**Pull request closed without merging**